Base/usr/share/man/man1/grep.md

```diff
@@ -5,7 +5,7 @@ grep
 ## Synopsis
 
 ```sh
-$ grep [--recursive] [--extended-regexp] [--regexp Pattern] [-i] [--invert-match] [--quiet] [--no-messages] [--binary-mode ] [--text] [-I] [--color WHEN] [file...]
+$ grep [--recursive] [--extended-regexp] [--regexp Pattern] [-i] [--invert-match] [--invert-match] [--quiet] [--no-messages] [--binary-mode ] [--text] [-I] [--color WHEN] [file...]
```

Comment:
```suggestion
$ grep [--recursive] [--extended-regexp] [--regexp Pattern] [-i] [--invert-match] [--line-numbers] [--quiet] [--no-messages] [--binary-mode ] [--text] [-I] [--color WHEN] [file...]
```

Also, should we maybe sort the flags up here (?) Also also, do intentionally only put the long forms up here?

Reply:
Regarding the copypasto: Will fix that

Regarding sorting: Meh, I don't really see the point.

Regarding long form: This file is [auto-generated through ArgsParser](https://github.com/SerenityOS/serenity/blob/066b1183cdd7dd0bd62bf0546cda7068946afcf2/Userland/Libraries/LibCore/ArgsParser.cpp#L267), and that code was mostly copied from the terminal formatter. Sure, that makes sense, but it's out of scope for this PR.

Now you know which code to change :)